### DISCUSSION TITLE
Missing Installation Step

### DISCUSSION THREAD

**Discussion by sfuertbauer** [bb-qq/r8152#396] [Ideas]
Please add the step to open Package Manager and start the Package "RTL8152/8153 Driver". Neither after installation nor after NAS restart, the package started automatically. Other than that, everything worked just fine (see my Show & Tell post)

**Comment by mackron** [comment]
This one got me today too, so +1 for this suggestion.

**Comment by bb-qq** [comment]
I added a step to start the package manually as suggested.

**Comment by DrMcCoy** [comment]
Okay, I just got my USB adapater, installed this package and it works like a dream. Thanks for that!

However, while after installing it, the driver run and worked no issues, I did restart my Synology box afterwards for DHCP lease reasons. And after a reboot, I had to manually start the driver.

So, question: will that now be necessary every time I reboot my Synology? I mean, I already planned to have the built-in port connected as a backup, and it's not like I reboot it often, but it would still be great if I didn't have to manually start the driver every time it does happen. Is that how this package is supposed to work or should it, theoretically, start on its own during boot-up?

I.e. should I try to investigate when I find the time and possibly create a bug ticket here, or should I look into if I can starting with a boot init script?

**Reply by bb-qq** [reply]
I do not assume that you have to start the driver each time manually.

Is it the same with a static IP address?

**Reply by castro145** [reply]
Howdy! Thank you very much for the driver, very much so appreciated. I ma having a same problem on DS918+, the package is not started.  I have a static address on it. After I stated the package, the NIC came up and the IP was still there, working like a charm.

**Reply by castro145** [reply]
I also have all of the logs if need to be, it almost seems as the system never even tries to start the package upon restart.

**Comment by castro145** [comment]
When I stop and start the package from dmseg

[ 1124.808910] r8152: module verification failed: signature and/or required key missing - tainting kernel
[ 1124.820527] usbcore: registered new interface driver r8152
[ 1125.023392] r8152 2-1:1.0 eth2: v2.18.1 (2024/05/20)
[ 1125.028973] r8152 2-1:1.0 eth2: This product is covered by one or more of the following patents:
[ 1125.046395] r8152 2-1:1.0 eth2: chip rev 14
[ 1128.645644] r8152 2-1:1.0 eth2: Promiscuous mode enabled
[ 1128.661031] r8152 2-1:1.0 eth2: carrier on
[ 1133.752524] r8152 2-1:1.0 eth2: Promiscuous mode enabled
[ 4981.660412] Module [r8152] is removed. 
[ 4981.664745] usbcore: deregistering interface driver r8152
[ 4988.065236] r8152 2-1:1.0 eth2: v2.18.1 (2024/05/20)
[ 4988.070818] r8152 2-1:1.0 eth2: This product is covered by one or more of the following patents:
[ 4988.088232] r8152 2-1:1.0 eth2: chip rev 14
[ 4988.092977] usbcore: registered new interface driver r8152
[ 4991.648388] r8152 2-1:1.0 eth2: Promiscuous mode enabled
[ 4991.654748] r8152 2-1:1.0 eth2: carrier on
[ 4994.943438] r8152 2-1:1.0 eth2: Promiscuous mode enabled
